docs/cli/tkn-results_config_set.md

```diff
@@ -7,74 +7,68 @@ Configure Tekton Results CLI settings
 Configure settings for the Tekton Results CLI.
 
 This command allows you to configure how the CLI interacts with the Tekton Results API server.
-It can automatically detect the API server in your cluster or allow manual configuration.
+It can automatically detect the API server in OpenShift environments or allow manual configuration.
 
 The command will:
-1. Automatically detect the Tekton Results API server in your cluster
+1. Automatically detect the Tekton Results API server in OpenShift environments
 2. Prompt for any missing configuration values
 3. Save the configuration for future use
 
-Automatic Detection:
-- Cluster context and namespace
-- API server endpoint
-- Service account token (if available)
-
-Manual Configuration (if automatic detection fails):
-- API server host (e.g., http://localhost:8080)
-- Authentication token
-- Additional cluster settings
-
-Configuration Options:
-  --host                    API server host URL
-  --token                   Authentication token
-  --api-path                API server path prefix
-  --insecure-skip-tls-verify Skip TLS certificate verification
-  --kubeconfig, -k          Path to kubeconfig file
-  --context, -c             Kubernetes context to use
-  --namespace, -n           Kubernetes namespace
-
-Note: Interactive prompts will be skipped if any configuration flag (host, token, api-path, insecure-skip-tls-verify) is used.
+Detection Strategy:
+- OpenShift: Automatically detects routes in openshift-pipelines and tekton-results namespaces
+- Kubernetes: Manual configuration required (automatic detection not available)
 
 Examples:
-  # Configure with automatic detection and interactive prompts
+  # OpenShift: Automatic detection
   tkn-results config set
 
-  # Configure with specific parameters (no prompts)
-  tkn-results config set --host=http://localhost:8080 --token=my-token
+  # Kubernetes: Manual configuration (required)
+  tkn-results config set --host=<api-server-url> --token=<token> --api-path=<path>
 
-  # Configure with custom API path and namespace (no prompts)
-  tkn-results config set --api-path=/api/v1 --namespace=my-namespace
+  # Manual configuration with custom settings
+  tkn-results config set --host=<api-server-url> --token=<token> --api-path=/api/v1 --insecure-skip-tls-verify
 
-  # Configure with custom kubeconfig and context
-  tkn-results config set --kubeconfig=/path/to/kubeconfig --context=my-cluster
+Automatic Detection (OpenShift only):
+- Detects routes in openshift-pipelines and tekton-results namespaces
```

Comment:
The `tekton-results` namespace is not used by Openshift users generally; it is only in an Openshift Cluster when the admin has installed Tekton Results separately in a custom namespace.